GREEDY/View/MainScreen.cs

```diff
@@ -83,5 +89,18 @@ private void ItemList_CellContentClick()
         {
             
         }
+
+        private void button1_Click(object sender, EventArgs e)
+        {
+            ImageViewer viewer = new ImageViewer(); 
+            VideoCapture capture = new VideoCapture(); 
+            Application.Idle += new EventHandler(delegate (object senderis, EventArgs e1)
+            {  
+                viewer.Image = capture.QueryFrame(); 
+            });
+            viewer.ShowDialog(); 
+            viewer.Image.Save(ConfigurationManager.AppSettings["imagePath"]);
+            capture.Dispose();
+        }
```

Comment:
Maybe in the future you could add instant OCR for captured pictures without saving it (just so you dont have to choose the file you just photographed).

Also we could implement the cropping of the picture (zooming in on just the receipt, adjusting the brightness, everything to make ocr easier)